src/SASjs.ts

```diff
@@ -442,23 +412,7 @@ export default class SASjs {
    *
    */
   public getUserName() {
-    return this.userName
-  }
-
-  /**
-   * Returns the _csrf token of the current session for the API approach.
-   *
-   */
-  public getCsrfApi() {
-    return this.csrfTokenApi?.value
-  }
-
-  /**
-   * Returns the _csrf token of the current session for the WEB approach.
-   *
-   */
-  public getCsrfWeb() {
```

Comment:
@krishna-acondy Is there a way now to get the web csrf token? Since DC tests are depending on it.

Reply:
This has now been brought back as `getCsrfToken()` which takes an optional `type` parameter to specify either the file CSRF token or the general one.